lib/Core/Executor.cpp

```diff
@@ -1249,40 +1250,46 @@ void Executor::executeCall(ExecutionState &state,
           // Alignment requirements for scalar types is the same as their size
           if (argWidth > Expr::Int64) {
              size = llvm::RoundUpToAlignment(size, 16);
```

Comment:
In that case we probably ought to have a comment noting the relevant section of the AMD-64 SYSV ABI